apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -60,6 +60,28 @@ const getChapter = (category?: string): number | undefined => {
   return +chapter[1] - 1
 }
 
+const formatCrimeScenePlace = (
+  street?: string,
+  streetNumber?: string,
+  municipality?: string,
+) => {
+  if (!street && !municipality) {
+    return ''
+  }
+
+  // Format the street and street number
+  const formattedStreet =
+    street && streetNumber ? `${street} ${streetNumber}` : street
+
+  // Format the municipality
+  const formattedMunicipality =
+    municipality && street ? `, ${municipality}` : municipality
+
+  const address = `${formattedStreet || ''}${formattedMunicipality || ''}`
```

Comment:
Please consider using ?? instead of ||. I think that is more appropriate in this context.